command/login/login_test.go

```diff
@@ -143,7 +143,7 @@ func TestLoginCommand(t *testing.T) {
 
 		code := cmd.Run(args)
 		require.Equal(t, code, 1, "err: %s", ui.ErrorWriter.String())
-		require.Contains(t, ui.ErrorWriter.String(), "403 (ACL not found)")
+		require.Contains(t, ui.ErrorWriter.String(), "500 (auth method \"test\" not found)")
```

Comment:
I don't think we want a 500 response code for this. Either a 404 or 403 seems appropriate. 

There's some related context in https://github.com/hashicorp/consul/pull/8520#issuecomment-816010606 (and the associated issue).

Reply:
Totally agreed, didn't know how to do that but your suggestion works like a charm!